src/libraries/Microsoft.PowerFx.Core/Public/Config/SymbolTable.cs

```diff
@@ -291,7 +298,7 @@ internal void AddEntity(IExternalEntity entity, DName displayName = default)
         /// <param name="type">Type of the object.</param>
         /// <param name="getValue">Call back that will retrieve object from the service provider.
         /// It can throw CustomFunctionErrorException, that fx will convert to an error.</param>
-        public void AddHostObject(string name, FormulaType type, Func<IServiceProvider, FormulaValue> getValue)
+        public void AddHostObject(string name, FormulaType type, Func<IServiceProvider,  Task<FormulaValue>> getValue)

```

Comment:
>   [](http://example.com/codeflow?start=87&length=2)

[minor] two spaces

Reply:
it hit right on my editor's wordwrap boundary - although I thought stylecop would catch these.